.github/workflows/dev.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       - run: pip install -U setuptools
       - run: pip install .
       - run: adt test_config.yaml --upload --platform GITHUB --run_id ${{ github.run_id }} --token ${{secrets.SYNAPSE_PAT}}
-      - run: adt modelad_test_config.yaml --platform GITHUB --run_id ${{ github.run_id }} --token ${{secrets.SYNAPSE_PAT}}
+      - run: adt modelad_test_config.yaml --upload --platform GITHUB --run_id ${{ github.run_id }} --token ${{secrets.SYNAPSE_PAT}}
```

Comment:
I'd prefer not to upload these reports from CI, as it runs too frequently.

I want the reports from the Seqera agora-nf runs to be uploaded.

Reply:
Yeah, that makes sense. But the seqra platform is currently using the model ad config in this repo: https://github.com/Sage-Bionetworks-Workflows/nf-agora/blob/main/nextflow.config#L33C23-L33C119 I will create two separate configs then.

Reply:
Let's see if the run I just kicked off uploads reports before we change anything on seqera: https://tower.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/agora-project/watch/1HioEVMC2RCm2Z

Reply:
I think it's probably already enabled there, since we do upload reports for other modelad datasets already.

Reply:
We can merge this & test Seqera once the upload flag is removed from the CI command above.